tests/Unit/SerializationTest.php

```diff
@@ -16,9 +16,7 @@ public function testSimpleEntityToSearchableArray(): void
     {
         $post = new Post('a simple post', 'some text', $datetime = new \DateTimeImmutable('@1728994403'));
         $idReflection = (new \ReflectionObject($post))->getProperty('id');
-        if (PHP_VERSION_ID < 80000) {
-            $idReflection->setAccessible(true);
-        }
+        $idReflection->setAccessible(true);
```

Comment:
why changed this? this was done on purpose, to be able to spot that this will need to be removed when php <8 won't be supported anymore by this lib :)

Reply:
It was a mistake. Reverted the change!